### DISCUSSION TITLE
RimeWiki 如何参与修改?

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1243] [Q&A] [answered]
例如当我发现 [wiki中一处](https://github.com/rime/home/wiki/CustomizationGuide#%E4%B8%80%E4%BE%8B%E5%AE%9A%E8%A3%BD%E6%A8%99%E9%BB%9E%E7%AC%A6%E8%99%9F) 有错误. 

> 
> ```
> 查看 default.yaml ，確有如下符號表：
>
>punctuator:
>  full_shape:
>    # ……其他……
>    "/" : [ ／, "/", ÷ ]
>    # ……其他……
>    half_shape:
>    # ……其他……
>    "/" : [ "/", ／, ÷ ]
>    # ……其他……
>```
> 

如何参与修改?

**Comment by lotem** [comment, marked as answer]
改好了。

wiki 設置爲只讓團隊成員修改。
之前發生用戶按照自己想法修改，沒有經過審核的情況。不好管理。

**Reply by pdog18** [reply]
> 改好了。
> 
> wiki 設置爲只讓團隊成員修改。 之前發生用戶按照自己想法修改，沒有經過審核的情況。不好管理。

[一例、定製喚出方案選單的快捷鍵](https://github.com/rime/home/wiki/CustomizationGuide#%E4%B8%80%E4%BE%8B%E5%AE%9A%E8%A3%BD%E5%96%9A%E5%87%BA%E6%96%B9%E6%A1%88%E9%81%B8%E5%96%AE%E7%9A%84%E5%BF%AB%E6%8D%B7%E9%8D%B5)
中的  `按鍵名稱 [參考這裏](https://github.com/rime/librime/blob/master/thirdparty/include/X11/keysymdef.h)` ，链接已经失效了。

**Reply by MaxHasBeenUsed** [reply]
**希望能在Discussion（或其他地方）開專區用於反饋 wiki 中的問題。**

~我想反饋一個：[這裏](https://github.com/rime/home/wiki/SpellingAlgebra#%E6%8B%BC%E5%AF%AB%E9%81%8B%E7%AE%97%E7%9A%84%E9%81%8B%E7%AE%97%E5%AD%90)的~

~> 實例二：算式 `derive/^[nl](.*)$/l$1/`  運算元 na  結果 la~
~> 效果：輸入la可獲得源碼表中與編碼na、la對應的候選；輸入na，候選仍爲碼表中編碼爲na的候選~
 
~似乎應該是~

~> 實例二：算式 `derive/^[nl](.*)$/l$1/`  運算元 na  結果 la~
~> 效果：輸入na可獲得源碼表中與編碼na、la對應的候選；輸入la，候選仍爲碼表中編碼爲la的候選~

edit: 实验了一下，原文确实没错。我再研究一下

**Reply by lotem** [reply]
> **希望能在Discussion（或其他地方）開專區用於反饋 wiki 中的問題。**
> 
> 我想反饋一個：[這裏](https://github.com/rime/home/wiki/SpellingAlgebra#%E6%8B%BC%E5%AF%AB%E9%81%8B%E7%AE%97%E7%9A%84%E9%81%8B%E7%AE%97%E5%AD%90)的
> 
> > 實例二：算式 `derive/^[nl](.*)$/l$1/`  運算元 na  結果 la
> > 效果：輸入la可獲得源碼表中與編碼na、la對應的候選；輸入na，候選仍爲碼表中編碼爲na的候選
> 
> 似乎應該是
> 
> > 實例二：算式 `derive/^[nl](.*)$/l$1/`  運算元 na  結果 la
> > 效果：輸入na可獲得源碼表中與編碼na、la對應的候選；輸入la，候選仍爲碼表中編碼爲la的候選

原文沒有錯。

**Comment by yurenchen000** [comment]
@lotem 
您好, 关于 ubuntu20,22 上 ibus 配置, 我也补充了一点
//这个系统上不操作这一步 (像原来文档中只用 ibus-setup 配置) 会找不到输入法

https://github.com/yurenchen000/rime_wiki/commit/63f8c0744

```diff
diff --git a/RimeWithIBus.md b/RimeWithIBus.md
index 0db8498..e5a210a 100644
--- a/RimeWithIBus.md
+++ b/RimeWithIBus.md
@@ -47,6 +47,13 @@ Rime 已收錄於 [Ubuntu 12.10 (Quantal Quetzal)](http://old-releases.ubuntu.co
     # 中古漢語拼音
     sudo apt-get install librime-data-zyenpheng
 
+启用输入法:
+
+   Settings > Keyboard > Input Sources, Click `+`
+     Select `Chinese` > `Chinese(Rime)`, Click `Add`
+
+  Note: `ibus-setup` add rime not work on Ubuntu.
+
 ## Fedora 22+
 
     sudo dnf install ibus-rime
```

**Reply by lotem** [reply]
随着软件版本的演进，启用输入法的方式略有变化。彼时，ibus的设置还没有集成到gnome，因此需要在ibus自己的设置界面添加。
